src-test/org/etools/j1939_84/controllers/part04/Part04Step13ControllerTest.java

```diff
@@ -165,16 +165,16 @@ public void testHappyPathNoFailures() {
         verify(verifier).verifyDataNotErased(any(), eq("6.4.13.4.a"));
 
         String expected = "";
-        expected += "Step 4.13.1.b. Waiting 5 seconds" + NL;
```

Comment:
We probably need to go back over the entire project and make sure we are consistent about the text we are putting into the progress bar. (no action required here)

Reply:
Already doing it as I see it.